eng/packaging.targets

```diff
@@ -1,16 +1,57 @@
 <Project InitialTargets="_OverridePackDependsOnForCsProjToPkgProj">
   
   <PropertyGroup Condition="'$(IsPackable)' == 'true'">
-    <TargetsForTfmSpecificContentInPackage>$(TargetsForTfmSpecificContentInPackage);AddRuntimeSpecificAssemblies;LibIntellisenseDocs</TargetsForTfmSpecificContentInPackage>
-    <TargetsForTfmSpecificDebugSymbolsInPackage>$(TargetsForTfmSpecificDebugSymbolsInPackage);AddRuntimeDebugSymbolsWithTfm</TargetsForTfmSpecificDebugSymbolsInPackage>
+    <!-- TODO: Remove when the SDK is upgraded to RC1. -->
+    <DotNetPackageValidationAssembly Condition="'$(MSBuildRuntimeType)' != 'Core'">$(NuGetPackageRoot)microsoft.dotnet.packagevalidation\$(MicrosoftDotNetPackageValidationVersion)\tools\net472\Microsoft.DotNet.PackageValidation.dll</DotNetPackageValidationAssembly>
```

Comment:
Why do we need to set these? the package should already set these correctly.

Reply:
When using the existing package the targets file complains that the property is empty in the UsingTasks statement. This doesn't happen anymore with the recent package. I won't merge this PR in until I updated the package tomorrow morning so this will go away.

Reply:
Sounds good!